demos/real_time_stream_analysis/python/README.md

```diff
@@ -130,8 +130,6 @@ __Console output:__
 2022-07-25 10:50:09,470 - stream_analyzer - INFO - For 100 received frames - FPS: 24; Frames dropped: 0;
 ```
 
-![minimal example](assets/minimal_example.jpg)
```

Comment:
Why are we removing console screenshots here?

Reply:
I added expected output in ```bash``` part so stdout from python3 real_time_stream_analysis.py command could be compared to it. It was doubled in console screenshots, so I removed it.